src/Apps/Artist/Components/ArtistHeader/ArtistHeader2.tsx

```diff
@@ -285,4 +291,11 @@ const CV = styled(RouterLink)`
   }
 `
 
+// remove paragraph margins from markdown
```

Comment:
This is "marked" doing and I'm afraid hardcoding this setting in MP that will produce breaking changes. Open for suggestions @dzucconi @jonallured

Reply:
We wouldn't want to hardcode this setting. This is normal and fine. Theoretically other tags with default margins could show up here but this is fine for now.